ev3/legoev3_motor.c

```diff
@@ -57,12 +57,11 @@
 #include "../motors/ev3_motor.h"
 
 #define TACHO_MOTOR_POLL_NS	2000000	/* 2 msec */
+#define TACHO_MOTOR_POLL_MS	2000	/* 2 msec */
```

Comment:
One is the tacho motor poll time in nanoseconds, the other is in milliseconds - I think that where they are used use differennt time scales - I'll eliminate one and do the math in-place on the other